### ISSUE TITLE
Possible inconsistency between tskit & tsinfer subcommands

### ISSUE THREAD

**Issue by hyanwong** [tskit-dev/tsinfer#274]
To get some information from a tree sequence file, I can do

```
python -m tskit info my_file_name.ts
```

but for a sample data file, to get information, I *list* it instead:

```
python -m tsinfer list my_file_name.samples
```

Could we possibly have an "info" option for tsinfer, either the same as `list` or a slimmed down version, so we at least have a consistent option for inspecting a `.ts` or `.samples` file?

**Comment by jeromekelleher** [comment]
Sure - ``info`` is probably better than ``list``. Could make the two synonymous initially with ``list`` being deprecated.

**Comment by hyanwong** [comment]
As [discussed](https://github.com/tskit-dev/tsinfer/pull/266#issuecomment-661810935), I think that `tsinfer info` could be a little more involved than just listing the stored arrays, and break down the number of mutations & sites by the inference method (fitch_parsimony vs full). In this case, we could leave the `list|ls` version as-is, ,and implement a separate info command which gives a bit more detail. This seems the right way around to me - `list` simply lists the arrays, `info` gives extra useful *info*mation

**Comment by jeromekelleher** [comment]
Agreed, getting a quick breakdown of useful stats from ``info`` would be very useful.
